src/Sylius/Bundle/AddressingBundle/Resources/config/doctrine/model/Country.orm.xml

```diff
@@ -14,11 +14,14 @@
 <doctrine-mapping xmlns="http://doctrine-project.org/schemas/orm/doctrine-mapping">
 
     <mapped-superclass name="Sylius\Component\Addressing\Model\Country" table="sylius_country">
+        <indexes>
+            <index columns="iso_name" />
+        </indexes>
         <id name="id" column="id" type="integer">
             <generator strategy="AUTO" />
         </id>
 
-        <field name="isoName" column="iso_name" type="string" />
+        <field name="isoName" column="iso_name" type="string" length="2" />
```

Comment:
@umpirsky Maybe we shouldn't limit the length?

Both "US" and "USA" should be allowed.